solidity/contracts/bridge/Fraud.sol

```diff
@@ -368,8 +379,10 @@ library Fraud {
         BridgeState.Storage storage self,
         bytes calldata walletPublicKey,
         uint32[] calldata walletMembersIDs,
-        bytes32 sighash
+        bytes memory sha256preimage
```

Comment:
I think we could stick with a `sighash` here since we don't do a signature check in this function.

Reply:
I think it makes sense to have `sha256preimage` for the consistency reason you mentioned. The fraud notifier does provides `walletPublicKey`, `sha256preimage` to `submitFraudChallenge`, I think it makes sense to continue with the same set of parameters for `notifyFraudChallengeDefeatTimeout`.